python/src/aac/lang/context.py

```diff
@@ -0,0 +1,165 @@
+"""The Active Context maintains the highly-contextual AaC DSL."""
```

Comment:
I've moved the context to language_context. That being said, I don't quite understand the underlying concern, is this still something we need to discuss after the move to the `aac.lang.language_context.py`?

Reply:
The concern was that it was defined in two different files `_active_context.py` and `context.py`. If that's no longer the case, then this can be marked as resolved.